setup.py

```diff
@@ -123,6 +123,11 @@ def read(*names, **kwargs):
     ],
     python_requires='>=2.7,<3',    
     install_requires=[
+        # Hack to support pip 8 (for those poor sods forced to use ubuntu 16.04's system pip)
+        # See https://github.com/nexB/scancode-toolkit/issues/1463
+        'more_itertools <  6.0.0; python_version == "2.7"',
```

Comment:
"more_itertools " is written as "more-itertools"

Reply:
Curses!  :-)  Thanks.

Reply:
> Curses! :-) Thanks.

Is i am wrong?

Reply:
You're probably right, and I should learn to test my code better.